framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -428,6 +447,45 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
                 result.add(new Postcondition(expr, postcondAnno, anno));
             }
         }
+
+        return result;
+    }
+
+    /**
+     * Returns the set of repeatable postconditions.
+     *
+     * @param metaAnno set of post condition meta annotations
```

Comment:
This comment is wrong.  The type of `metaAnno` is `AnnotationMirror`, so it is not a set.  Please write accurate comments.  Do you mean that it is a wrapper for repeatable annotations?

Also, "postcondition" is one word (though you may use a hyphen if you prefer), and "meta-annotations" is hyphenated.

Reply:
I have updated the documentation but I am having trouble with this. Please help in this!!